database/migrations/2017_06_07_173437_add_multiple_genders_choices.php

```diff
@@ -13,8 +13,40 @@ class AddMultipleGendersChoices extends Migration
      */
     public function up()
     {
-        DB::statement("ALTER TABLE contacts CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
-        DB::statement("ALTER TABLE significant_others CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
-        DB::statement("ALTER TABLE kids CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
+        switch(DB::connection()->getPDO()->getAttribute(PDO::ATTR_DRIVER_NAME))
+        {
+            case 'mysql':
+                DB::statement("ALTER TABLE contacts CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
+                DB::statement("ALTER TABLE significant_others CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
+                DB::statement("ALTER TABLE kids CHANGE COLUMN gender gender ENUM('male', 'female', 'none')");
+                break;
+            case 'pgsql':
+                $this->alterEnum('contacts', 'gender', ['male', 'female', 'none']);
+                $this->alterEnum('significant_others', 'gender', ['male', 'female', 'none']);
+                $this->alterEnum('kids', 'gender', ['male', 'female', 'none']);
+                break;
+            default:
+                throw new \Exception('Driver not supported.');
```

Comment:
Would you put the driver name in this string, please?

It is easier to help someone with a message `Driver foobar is not supported`.